bin/stacks/routing-api-stack.ts

```diff
@@ -18,8 +18,10 @@ import { RoutingLambdaStack } from './routing-lambda-stack'
 import { RoutingDatabaseStack } from './routing-database-stack'
 import { RpcGatewayDashboardStack } from './rpc-gateway-dashboard'
 import { REQUEST_SOURCES } from '../../lib/util/requestSources'
+import { TESTNETS } from '../../lib/util/testNets'
 
-export const CHAINS_NOT_MONITORED: ChainId[] = [ChainId.GOERLI, ChainId.POLYGON_MUMBAI]
+export const CHAINS_NOT_MONITORED: ChainId[] = TESTNETS;
```

Comment:
Are we still using this const?

Reply:
yep, in https://github.com/Uniswap/routing-api/blob/main/bin/stacks/routing-api-stack.ts#L342 and  https://github.com/Uniswap/routing-api/blob/main/bin/stacks/routing-api-stack.ts#L377